packages/extension-chakra-storefront/src/setup-app.tsx

```diff
@@ -54,58 +54,57 @@ class ChakraStorefront extends ApplicationExtension<Config> {
 
         const extensionRoutes = [
             {
-                path: config.pages.Home,
+                path: config.pages.Home && config.pages.Home.path,
```

Comment:
Since we allow false | PageConfigs[K] for pageType. Typescript will complain if we don't check the page before looking for path.